release-notes/6.0/6.0.0.md

```diff
@@ -0,0 +1,139 @@
+
+## Target Platform Support
+- .NET Framework 4.6.2+ (Windows x64, Windows x86)
+- .NET 8.0, 9.0 (Linux, macOS, Windows ARM, Windows ARM64, Windows x64, Windows x86)
+
+### Dependencies
+
+#### .NET Framework
+- Azure.Identity 1.11.4
+- Microsoft.Bcl.Cryptography 9.0.0
+- Microsoft.Data.SqlClient.SNI 6.0.0-preview1.24226.4
+- Microsoft.Extensions.Caching.Memory 9.0.0
+- Microsoft.IdentityModel.JsonWebTokens 7.5.0
+- Microsoft.IdentityModel.Protocols.OpenIdConnect 7.5.0
+- System.Buffers 4.5.1
+- System.Security.Cryptography.Pkcs 9.0.0
+- System.Text.Encodings.Web 6.0.0
+- System.Text.Json 6.0.10
+
+#### .NET 8
+
+- Azure.Identity 1.11.4
+- Microsoft.Bcl.Cryptography 9.0.0
+- Microsoft.Data.SqlClient.SNI.runtime 6.0.0-preview1.24226.4
+- Microsoft.Extensions.Caching.Memory 9.0.0
+- Microsoft.IdentityModel.JsonWebTokens 7.5.0
+- Microsoft.IdentityModel.Protocols.OpenIdConnect 7.5.0
+- Microsoft.SqlServer.Server 1.0.0
+- System.Configuration.ConfigurationManager 9.0.0
+- System.Security.Cryptography.Pkcs 9.0.0
```

Comment:
I'm concerned that we are targeting the .NET 9 version of dependencies in our .NET 8 target. Since .NET 9 is not an LTS release and will go out of support before .NET 8, we could end up caught in a situation where we can't update a dependency version to address a vulnerability because .NET 9 won't be serviced for that vulnerability. (We also want to avoid bumping major versions of dependencies within a major version of MDS.) We should try to target LTS versions of dependencies from .NET, when possible.

I didn't catch the PR that bumped these, or I would have said something earlier.

Reply:
No need for a .net 9 version props, but the external packages should be LTS versions, imo

Reply:
@David-Engel I agree, we should be on a stable release for SNI. The whole thing kinda depends on it :) @cheenamalhotra is there anything I need to do for that?